src/test/kotlin/nl/info/zac/app/zaak/ZaakKoppelenRestServiceTest.kt

```diff
@@ -717,17 +913,17 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
 
             And("required services should be invoked") {
                 verify(exactly = 1) {
-                    zrcClientService.readZaak(sourceZaak.uuid)
+                    zrcClientService.readZaak(deelzaak.uuid)
```

Comment:
it seems this verification is done many times in this test, maybe it's better to extract this into it's own verification method?

Reply:
the whole block I mean, where you pass the expected values... 
something like this:


```kotlin
fun verifyServicesHaveBeenInvoked(
    exactly: Int, 
    readZaakUuid: UUID,
    readZaakRechtenZaak: Zaak,
    readZaakRechtenZoekObject: ZaakZoekObject ) {
    verify(exactly = exactly) {
        zrcClientService.readZaak(readZaakUuid)
        searchService.zoek(any())
        policyService.readZaakRechten(readZaakRechtenZaak)
        policyService.readZaakRechten(readZaakRechtenZoekObject)
    }
}
```